<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>indexWindow</class>
 <widget class="QMainWindow" name="indexWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(210, 210, 210,1);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(210, 210, 210,1);</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QPushButton" name="btn_index_qldvsx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
     </property>
     <property name="text">
      <string>Nơi sản xuất</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_index_qldvpp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);</string>
     </property>
     <property name="text">
      <string>Nơi phân phối</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_index_qlsp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);</string>
     </property>
     <property name="text">
      <string>Quản lý sản phẩm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_index_qlbh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);</string>
     </property>
     <property name="text">
      <string>Quản lý bán hàng</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_index_qlnv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);</string>
     </property>
     <property name="text">
      <string>Quản lý nhân viên</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_index_thongke">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(215, 220, 88, 0.77);</string>
     </property>
     <property name="text">
      <string>Thống kê</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(225, 130, 146, 1);</string>
     </property>
     <property name="text">
      <string>Đăng xuất</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/pic/nuoc-ngot-co-ga-1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Trang chủ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="im1.qrc"/>
 </resources>
 <connections/>
</ui>
